test/perf.jl

```diff
@@ -6,15 +6,22 @@ using LinearAlgebra
 using BenchmarkTools
 using SparseArrays
 using TimerOutputs
-import ExaPF: Parse, PowerSystem
+import ExaPF: ParsePSSE, PowerSystem, IdxSet
```

Comment:
Thanks for updating `test/perf.jl` as well!

Reply:
:)